docker_hub/README.md

```diff
@@ -0,0 +1,10 @@
+Use the Adapt Docker image as follows:
+
+1. Pull the adapt image to your machine: `docker pull adapt:latest`
+
+2. If you are using git, add this alias to your `.bashrc`:
```

Comment:
Looks like you updated the command to use $(pwd) instead. I think that's a good change. So I'd remove the reference to git here.

Reply:
Done plus I updated the docs a bit as well.